editor/editor_node.cpp

```diff
@@ -7361,7 +7361,9 @@ EditorNode::EditorNode() {
 	vcs_actions_menu = VersionControlEditorPlugin::get_singleton()->get_version_control_actions_panel();
 	vcs_actions_menu->set_name("Version Control");
 	vcs_actions_menu->connect("index_pressed", callable_mp(this, &EditorNode::_version_control_menu_option));
-	vcs_actions_menu->add_item(TTR("Create Version Control Metadata..."), RUN_VCS_METADATA);
+	Ref<FileAccess> fileAccess = FileAccess::create(FileAccess::ACCESS_RESOURCES);
+	String isGitInitialisedSymbol = (fileAccess->file_exists(current_path.path_join(".gitignore")) ? "Override" : "Create");
+	vcs_actions_menu->add_item(TTR(isGitInitialisedSymbol + " Version Control Metadata..."), RUN_VCS_METADATA);
```

Comment:
`TTR()` must contain a string literal, not an expression. Otherwise, our script will not be able to extract the string for translation.